R/autoplot.seroincidence.by.R

```diff
@@ -7,16 +7,17 @@
 #' @return an object of class `"ggarrange"`, which is a [ggplot2::ggplot()] or a [list()] of [ggplot2::ggplot()]s.
 #' @export
 #' @examples
-#' \dontrun{
+#'
 #' library(dplyr)
 #' library(ggplot2)
 #'
 #' xs_data <- "https://osf.io/download//n6cp3/" %>%
 #' load_pop_data() %>%
 #' clean_pop_data
 #'
-#' curve <- "https://osf.io/download/rtw5k/" %>%
-#' load_curve_params()
+#' curve = load_curve_params("https://osf.io/download/rtw5k/") %>%
+#' filter(antigen_iso == c("HlyE_IgA", "HlyE_IgG")) %>%
+#'   slice(1:100, .by=antigen_iso) #reduce dataset for the purposes of this example
```

Comment:
- [ ] Please format examples according to the style guide: https://style.tidyverse.org/syntax.html#infix-operators